packages/compass-import-export/src/constants/file-types.ts

```diff
@@ -1,4 +1,8 @@
-const FILE_TYPES = {
+export type AcceptedFileType = 'csv' | 'json';
+
+const FILE_TYPES: {
+  [AcceptedFileType: string]: string;
+} = {
```

Comment:
Just curious, this is making the type less specific than it previously was with just `as const` – any particular reason for that (now it’s `{ [key: string]: string }`, it used to be `{ CSV: 'csv', JSON: 'json' }`)?

Reply:
Nope, thanks for catching! Updated to be more strict:
```ts
const FILE_TYPES: {
  [fileType: string]: AcceptedFileType;
} = {
```
Eventually we'll remove this variable once everything is using the `AcceptedFileType`.